flashinfer/apply.py

```
@@ -0,0 +1,29 @@
+    ApplyError,
+    disable_apply,
+    enable_apply,
+)
+from .trace_apply.config import ApplyConfig
+
+__all__ = [
+    "ApplyError",
+    "ApplyConfig",
+    "enable_apply",
+    "disable_apply",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)